xml/snapper.xml

```diff
@@ -1961,18 +1981,18 @@ local  | /local</screen>
   <sect2 xml:id="sec-snapper-manage-create">
    <title>Creating Snapshots</title>
    <para>
-    Creating a snapshot is done by running <command>snapper create</command> or
-    by clicking <guimenu>Create</guimenu> in the &yast; module
+    To create a snapshot, run <command>snapper create</command> or
+    click <guimenu>Create</guimenu> in the &yast; module
     <guimenu>Snapper</guimenu>. The following examples explain how to create
-    snapshots from the command line. It should be easy to adopt them when using
+    snapshots from the command line. However, they are It should be easy to adapt them when using
```

Comment:
It is hard to tell in the Github interface but I think there may be some redundant text left here...? I will pull, build and check.

Reply:
Yep. Looks like:
> snapshots from the command line. However, they are It should be easy to adapt them when using

I think you want _either_

"However, they are easy to adapt when using"

or 

"It should be easy to adapt them when using"

Reply:
Yeah -- that looks wrong. I originally wanted to clean up the sentence properly but had no good ideas, so I tried to settle for just the adopt/adapt fix. Will fix tomorrow.